facturacion_mexico/cfdi_recibidos/queries.py

```
@@ -0,0 +1,98 @@
+import frappe
+
+from facturacion_mexico.cfdi_recibidos.services.uom_policy import get_sat_uom_list
+
+
+@frappe.whitelist()
+@frappe.validate_and_sanitize_search_inputs
+def get_expense_item_groups(doctype, txt, searchfield, start, page_len, filters):
```

Ruff (0.15.14)

[warning] Unused function argument: `doctype`

(ARG001)